src/components/Todo/Todo.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Todo.scss';
+import User from '../User/User';
+
+function Todo({ id, title, completed, user }) {
+  return (
+    <div className="card">
+      <p>{id}</p>
+      {title}
+      <User {...user} />
+      {completed
+        ? <p className="active">Done</p>
+        : <p className="disable">Not done</p>
```

Comment:
```suggestion
      <p className={completed ? 'active' : 'disable'}>
        {completed ? 'Done' : 'Not done'}
      </p>
```

і ще є така ліба для класів
https://www.npmjs.com/package/classnames